src/sidebar/components/hooks/test/use-root-thread-test.js

```diff
@@ -19,22 +22,26 @@ describe('sidebar/components/hooks/use-root-thread', () => {
       '../../store/use-store': callback => callback(fakeStore),
       '../../util/thread-annotations': fakeThreadAnnotations,
     });
+
+    // Mount a dummy component to be able to use the `useRootThread` hook
+    // Do things that cause `useRootThread` to recalculate in the store and
+    // test them (hint: use `act`)
+    function DummyComponent() {
+      useRootThread();
+    }
+    mount(<DummyComponent />);
   });
 
   afterEach(() => {
     $imports.$restore();
   });
 
   it('should return results of `threadAnnotations` with current thread state', () => {
-    fakeThreadAnnotations.returns('a tisket, a tasket');
-
-    const results = useRootThread();
-
     const threadState = fakeThreadAnnotations.getCall(0).args[0];
```

Comment:
This test now checks that `threadAnnotations` was called, but not that `useRootThread` returned the result of that call. I would suggest to do something like:

```js
let rootThread;

function DummyComponent() {
  rootThread = useRootThread();
}
mount(<DummyComponent />);

// Assert on `rootThread` here
```

Reply:
Updated.